cob_hardware_config/robots/cob4-7/urdf/cob4-7.urdf.xacro

```diff
@@ -24,6 +24,7 @@
 
   <!-- sensors -->
   <xacro:include filename="$(find cob_description)/urdf/sensors/realsense.urdf.xacro" />
+  <xacro:include filename="$(find cob_description)/urdf/sensors/realsense_zr300.urdf.xacro" />
```

Comment:
this file does not exist yet! the PR to cob_common is missing!

BTW, what's the difference between the two urdf descriptions? if there is none (or it is minimal), I'd prefer to have both realsense_cameras in the same xacro....(and maybe use some xacro arguments)